packages/hooks/src/useAvailableWallets.ts

```diff
@@ -8,12 +8,14 @@ const useAvailableWallets = () => {
     const { client, ui } = useStore();
     const { is_dark_mode_on } = ui;
     const { is_crypto } = client;
-    const { data } = useAuthorize();
+    const { data, isSuccess: is_authorize_success, isFetching: is_authorize_fetching } = useAuthorize();
```

Comment:
@markwylde-deriv `isSuccess` and `isFetching` are just flags from `TanStack Query`. I renamed it to do it more understandable. In my opinion `is_authorize_success` is more clear name than `isSuccess`. Next in `useWalletsList` I return it like this because we need to return whole object with all flags, but rest doesn't have it
```
{
        ...rest,
        isSuccess: is_authorize_success,
        isFetching: is_authorize_fetching,
        data: sortedWallets,
    }
```

Reply:
@markwylde-deriv I could use `rest.isSuccess` and `rest.isFetching` and return will be the same as before, but I think there is no difference

Reply:
I mean, why call it isSuccess at all. Can we do:

```
    {
        ...rest,
        is_authorize_success,
        is_authorize_fetching,
        data: sortedWallets,
    }
```

Reply:
@markwylde-deriv I suppose no, because we as developers expect to receive all options from `TanStack Query` with the same names. [Link](https://tanstack.com/query/v4/docs/react/reference/useQuery) to the doc